pkg/objects/bridge/grpc.go

```diff
@@ -17,17 +17,16 @@ import (
 
 	pb "github.com/opiproject/opi-api/network/evpn-gw/v1alpha1/gen/go"
 
-	"go.einride.tech/aip/fieldbehavior"
 	"go.einride.tech/aip/resourceid"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
-	"google.golang.org/protobuf/types/known/emptypb"
 )
 
-// CreateLogicalBridge executes the creation of the LogicalBridge
-func (s *Server) CreateLogicalBridge(ctx context.Context, in *pb.CreateLogicalBridgeRequest) (*pb.LogicalBridge, error) {
+// Create executes the creation of the LogicalBridge
```

Comment:
if this function operates on models now, where in the code we no return grpc response ?

Reply:
Good point @glimchb. I've pulled back the abstraction to focus on consolidating how the operations are performed on the store. At the boundaries of each pb operation proto bufs are used, but the internal workings of those functions are consolidated to a set of store operations defined in `objects/object.go`